hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/query/QueryWorkflowImplTest.java

```diff
@@ -69,1059 +69,428 @@
 
 @ExtendWith(MockitoExtension.class)
 class QueryWorkflowImplTest extends AppTestBase {
-//
```

Comment:
I'm worried about this one too. At least include the commented out code + the new code and I can sort it out.

Reply:
Reverted this to `04712-use-pbj` contents.